config/default/views.view.blog_rss.yml

```
@@ -0,0 +1,259 @@
+          value:
+            1: 1
+          group: 1
+          exposed: false
+          expose:
+            operator_id: tid_op
+            label: Topics
+            description: ''
+            use_operator: false
+            operator: tid_op
+            operator_limit_selection: false
+            operator_list: {  }
```

YAMLlint (1.38.0)

[error] too many spaces inside empty braces

(braces)